frontend/src/scenes/plugins/PluginsSearch.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { pluginsLogic } from 'scenes/plugins/pluginsLogic'
+import { Input } from 'antd'
+
+export function PluginsSearch(): JSX.Element {
+    const { searchTerm, rearranging } = useValues(pluginsLogic)
+    const { setSearchTerm } = useActions(pluginsLogic)
+    return (
+        <Input
+            placeholder="Start typing to search for a plugin"
+            onChange={(e) => setSearchTerm(e.target.value)}
+            style={{ maxWidth: 400, height: 40, borderRadius: 5, marginTop: 5, marginBottom: 10 }}
```

Comment:
This seems kinda bulky to me. See person search:
![1](https://user-images.githubusercontent.com/4550621/125701750-37c531ed-7157-4958-9ab5-091ccc56d567.png)
vs. plugin search
![2](https://user-images.githubusercontent.com/4550621/125701753-9089d823-1a3b-43d2-a783-12a3c53a0e91.png)

Reply:
Personal preference - I like bulky (see https://posthog.com/contributors)

Will update for consistency though